samples/xamarin-forms/Cinephile/Cinephile.ViewModels/ViewModelBase.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reactive;
+using System.Reactive.Concurrency;
+using ReactiveUI;
+using Splat;
+
+namespace Cinephile.ViewModels
+{
+    public abstract class ViewModelBase : ReactiveObject, IRoutableViewModel, ISupportsActivation
+    {
+        public string UrlPathSegment
+        {
+            get;
+            private set;
```

Comment:
Not sure about these two `private set`-ters also, probably can be made `get`-only as well :)

Reply:
And you are right again :)